runtime/web_worker.rs

```diff
@@ -325,10 +327,17 @@ impl WebWorker {
     });
 
     let inspector = if options.attach_inspector {
-      Some(DenoInspector::new(
-        &mut js_runtime,
-        options.maybe_inspector_server.clone(),
-      ))
+      let (new_websocket_tx, new_websocket_rx) =
+        mpsc::unbounded::<WebSocketProxy>();
+
+      let inspector = DenoInspector::new(&mut js_runtime, new_websocket_rx);
+
+      if let Some(server) = options.maybe_inspector_server.clone() {
+        let info = InspectorInfo::new(server.host, new_websocket_tx);
+        server.register_inspector(info);
+      }
+
+      Some(inspector)
```

Comment:
Hopefully this bit can be refactored a bit more once I decouple inspector from `WebSocketSession` abstraction